### ISSUE TITLE
nnCron and Github 

### ISSUE THREAD

**Issue by VoidVolker** [ForthHub/discussion#16]
nnCron — most powerful automation program for windows now available on github: https://github.com/nnCron/nnCron
nnCron here is only one day and I'm still work on repositories filling. Any suggestions, bug reports and plugins are welcome.

**Comment by larsbrinkhoff** [comment]
Nice to see a serious Forth application!

**Comment by VoidVolker** [comment]
Thanks ) nnCron is very old, but still very powerful. Do you use it?

**Comment by hcchengithub** [comment]
Just read the README.md, Wow! will try.

On Thu, Nov 19, 2015 at 3:19 AM, VoidVolker <notifications@github.com>
wrote:

> Thanks ) nnCron is very old, but still very powerful. Do you use it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ForthHub/discussion/issues/16#issuecomment-157828053>.
>



-- 
Just undo it!
陳厚成 H.C. Chen
Nothing real can be threatened. Nothing unreal exists. Herein lies the
peace of God.
JavaScript Forth http://github.com/hcchengithub/jeforth.3we

**Comment by hcchengithub** [comment]
nnCorn console doesn't work on my Windows 10.
Did I do it correctly?


​
​

On Thu, Nov 19, 2015 at 12:04 PM, H.C. Chen <hcchen5600@gmail.com> wrote:

> Just read the README.md, Wow! will try.
>
> On Thu, Nov 19, 2015 at 3:19 AM, VoidVolker <notifications@github.com>
> wrote:
>
>> Thanks ) nnCron is very old, but still very powerful. Do you use it?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/ForthHub/discussion/issues/16#issuecomment-157828053>
>> .
>>
>
>
>
> --
> Just undo it!
> 陳厚成 H.C. Chen
> Nothing real can be threatened. Nothing unreal exists. Herein lies the
> peace of God.
> JavaScript Forth http://github.com/hcchengithub/jeforth.3we
>
>


-- 
Just undo it!
陳厚成 H.C. Chen
Nothing real can be threatened. Nothing unreal exists. Herein lies the
peace of God.
JavaScript Forth http://github.com/hcchengithub/jeforth.3we

**Comment by VoidVolker** [comment]
nnCron is use by default 2002 port for console - is it open? Check your firewall. Also, for OS from Vista and later nnCronn better to run as application, not service.

**Comment by hcchengithub** [comment]
I am still in the dark to make a quick start. ( on PC with Windows 10 )
Run startnncron.bat is read the easiest way, a service named nnCron is
really running then.
But there's no nnCron tray icon as said, blocking the Lesson-1 in help.chm
When trying to use Forth console that is supposed to be interesting.
But as mentioned it stopped at the error.
I don't know how to open a port neither dealing with the firewall. It seems
too much only to see the Forth "OK" prompt.
Can't find how to run as an application either.








On Thu, Nov 19, 2015 at 3:06 PM, VoidVolker <notifications@github.com>
wrote:

> nnCron is use by default 2002 port for console - is it open? Check your
> firewall. Also, for OS from Vista and later nnCronn better to run as
> application, not service.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ForthHub/discussion/issues/16#issuecomment-157971964>.
>



-- 
Just undo it!
陳厚成 H.C. Chen
Nothing real can be threatened. Nothing unreal exists. Herein lies the
peace of God.
JavaScript Forth http://github.com/hcchengithub/jeforth.3we

**Comment by VoidVolker** [comment]
You need install nnCron as application, not as service in windows 10. https://github.com/nnCron/nnCron/wiki/FAQ#1
Windows firewall will ask permission to open ports on nnCron first run. If you use other firewall - you can do it via instruction for this firewall.

**Comment by hcchengithub** [comment]
Q: How to run nnCron as an application?
A: command line : nncron.exe -ns
Shooooo.... It works on Windows 10 now. The tray icon finally appears.
Thank you very much.

**Comment by VoidVolker** [comment]
Yes, correct. Also, you can choose this option when install nncron. Thanks for feedback, FAQ is updated.
